deps/fs.lua

```diff
@@ -577,3 +577,30 @@ end
 function fs.createReadStream(path, options)
   return fs.ReadStream:new(path, options)
 end
+function fs.appendFile(filename, data, callback)
+  callback = callback or function() end
+  local function write(fd, offset, buffer, callback)
+    local function onWrite(err, written)
+      if err then return fs.close(fd, function() callback(err) end) end
+      if written == #buffer then
```

Comment:
This is still going to be a problem with partial writes since `#buffer` is always the full buffer, but you only write part in the second write.